### PR TITLE
removed unused assignments. Sometimes it allows removing related blocks

### PR DESCRIPTION
**Pull request by dk2k**
# Description
removed unused assignments. Sometimes it allows removing related blocks of code

# Target

`main` branch

# Tracking
no dedicated ticket

# Apache Camel coding standards and style

- [x] I checked that each commit in the pull request has a meaningful subject line and body.
- [x] I formatted the code using `mvn -Pformat,fastinstall install && mvn -Psourcecheck`

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: components/camel-sjms/src/test/java/org/apache/camel/component/sjms/support/SjmsConnectionTestSupport.java [hunk 1]
```diff
@@ -55,8 +55,6 @@ public void setup() throws Exception {
                 || getConnectionUri().startsWith("vm")) {
             vmTestConnectionFactory = new ActiveMQConnectionFactory(
                     VM_BROKER_CONNECT_STRING);
-        } else {
-            createBroker();
```
Comment (davsclaus) [inline components/camel-sjms/src/test/java/org/apache/camel/component/sjms/support/SjmsConnectionTestSupport.java:59]: Is this correct ?

Comment (orpiske) [reply components/camel-sjms/src/test/java/org/apache/camel/component/sjms/support/SjmsConnectionTestSupport.java:59]: @davsclaus in theory no, but I think this class is unused. 

I think the whole class can be removed. It does not seem to be used anywhere else. Please, can you give it a try @dk2k ?

Comment (dk2k) [reply components/camel-sjms/src/test/java/org/apache/camel/component/sjms/support/SjmsConnectionTestSupport.java:59]: Yes. If I remove unused assignment in createBroker(), then I can remove all it's code. Eventually I can delete the method and its call

Comment (orpiske) [reply components/camel-sjms/src/test/java/org/apache/camel/component/sjms/support/SjmsConnectionTestSupport.java:59]: @dk2k I triggered a new build, but I don't think it's related to your changes.

**Review by orpiske — APPROVED** [review]
I reviewed the test failure and it's on core (so, unrelated to this). We should be good to merge.

**Pull request merged**